spring-cloud-dataflow-configuration-metadata/src/test/java/org/springframework/cloud/dataflow/configuration/metadata/BootApplicationConfigurationMetadataResolverTests.java

```diff
@@ -53,12 +54,19 @@ class BootApplicationConfigurationMetadataResolverTests {
 
 	private ApplicationConfigurationMetadataResolver resolver;
 
+	private AutoCloseable autoCloseable;
+
 	@BeforeEach
 	void init() {
-		MockitoAnnotations.initMocks(this);
+		autoCloseable = MockitoAnnotations.openMocks(this);
```

Comment:
**[UNRELATED]** This was already here I know. The usage of `@Mock` has turned out to be more hassle than its worth. Like in this case, its a single mock and we have this setup/teardown to deal w/.